### PR TITLE
Documenting small things here and there..

### PR DESCRIPTION
**Pull request by CGMossa**
Hey! I've just started using Bevy by following [Snake tutorial](https://mbuffett.com/posts/bevy-snake-tutorial/) and I'm just tinkering with documenting some things here and there. 

Please provide feedback.

### CODE REVIEW

File: crates/bevy_app/src/app_builder.rs [hunk 1]
```diff
@@ -220,6 +220,8 @@ impl AppBuilder {
             .add_system_to_stage(stage::EVENT_UPDATE, Events::<T>::update_system.system())
     }
 
+    /// Adds a resource to the current [`App`], and overwritting any resource
```
Comment (cart) [inline crates/bevy_app/src/app_builder.rs:223]: nit: lets make this

>  Adds a resource to the current [`App`] and overwrites any resource previously added of the same type.

File: crates/bevy_asset/src/loader.rs [hunk 2]
```diff
@@ -168,7 +168,7 @@ impl<T: Resource> Default for AssetLifecycleChannel<T> {
     }
 }
 
-/// Reads [AssetResult]s from an [AssetChannel] and updates the [Assets] collection and [LoadState] accordingly
+/// Reads [AssetResult]s from an [AssetLifecycleChannel] and updates the [Assets] collection and [crate::LoadState] accordingly
```
Comment (cart) [inline crates/bevy_asset/src/loader.rs:171]: This is still true, but it is now a subset of what is done. I think this should probably say something a bit more generic like:

> Updates the [Assets] collection according to the changes queued up by [AssetServer]

File: crates/bevy_app/src/app_builder.rs (hunk 1 above)
Comment (cart) [reply crates/bevy_app/src/app_builder.rs:223]: I also think it should be `[App]` not ``[`App`]``

**Comment by CGMossa** [comment]
Thanks for the additions. I'll push them in a second.

The CI fails if my file isn't `rustfmt`'d, but if a I run `cargo fmt` inside of `bevy`, then all the
files gets modified.. I guess, should I push a commit that has formatted every bevy file using `cargo fmt`?

File: crates/bevy_asset/src/loader.rs (hunk 2 above)
Comment (CGMossa) [reply crates/bevy_asset/src/loader.rs]: Added this change.

**Comment by CGMossa** [comment]
> The CI fails if my file isn't `rustfmt`'d, but if a I run `cargo fmt` inside of `bevy`, then all the
> files gets modified.. I guess, should I push a commit that has formatted every bevy file using `cargo fmt`?

Never mind. I figured that it was a windows thing. I am using different line endings than `Unix`. I changed
`rustfmt.toml` line to `newline_style = "Native"`, and that worked fine. I won't commit this change.. 
But these line endings might come from GIT, maybe?

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Code of Conduct
 
-_This Code of Conduct is based on the [Rust Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct), which is adapted from the [Node.js Policy on Trolling](http://blog.izs.me/post/30036893703/policy-on-trolling) and the [Contributor Covenant](https://www.contributor-covenant.org)._
+_This Code of Conduct is based on the [Rust Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct), which is adapted from the [Node.js Policy on Trolling](http://blog.izs.me/post/30036893703/policy-on-trolling) and the [Contributor Covenant](https://www.contributor-covenant.org).
```
Comment (cart) [inline CODE_OF_CONDUCT.md:3]: This breaks the italic formatting. We should re-add the `_`

Comment (CGMossa) [reply CODE_OF_CONDUCT.md]: Ups. Fixed. Thanks

**Pull request merged**